src/utils/time.ts

```diff
@@ -0,0 +1,4 @@
+export const addLeadingZero = (relativeTime: number): string =>
+  relativeTime >= 0 && relativeTime <= 9
+    ? `0${relativeTime}`
+    : relativeTime.toString()
```

Comment:
I would name this file as a `general` or `number` utils. I mean we could add leading zeros for numbers in different contexts not just in terms of time.

Reply:
Fixed in ec8215de3633e25016000ed44a9423ff9cc7672b